.github/actions/bug-refs/index.js

```diff
@@ -0,0 +1,61 @@
+const core = require('@actions/core');
+const github = require('@actions/github');
+
+async function run() {
+    const context = github.context;
+    if (context.eventName !== "pull_request") {
+      console.log(
+        'The event that triggered this action was not a pull request, skipping.'
+      );
+      return;
+    }
+
+    const body = context.payload.pull_request.body;
+    const ignoreJira = body.toLocaleUpperCase().includes("NO-JIRA");
+    const ignoreLaunchpad = body.toLocaleUpperCase().includes("NO-LP");
+    const ignoreGithub = body.toLocaleUpperCase().includes("NO-GH");
+
+    const errorMessages = [];
+
+    const title = context.payload.pull_request.title;
+    if (!ignoreJira && !title.toLocaleUpperCase().includes("SC-")) {
+        errorMessages.push("The PR title does not include a Jira SC-#### reference.\nEither add one or add 'no-jira' to the PR description.");
+    }
+
+    const client = github.getOctokit(
+        core.getInput('repo-token', {required: true})
+    );
+    const commits = await client.rest.pulls.listCommits({
+        owner: context.issue.owner,
+        repo: context.issue.repo,
+        pull_number: context.issue.number,
+    });
+
+    const launchpadPresent = false;
+    const githubPresent = false;
+    commits.data.forEach(commit => {
+        const message = commit.commit.message;
+        if (message.toLocaleUpperCase().includes("LP: #")) {
+            launchpadPresent = true;
+        }
+        if (message.toLocaleUpperCase().includes("FIXES: #")) {
```

Comment:
Per the release documentation, I think we use `log2dch`, which collects GitHub's issues referenced as `GH: #NNNN`: https://github.com/canonical/uss-tableflip/blob/3c02cd907c2c5bb799b6c88926e041b2e91f4069/scripts/log2dch#L218

Shouldn't we use and check for that references instead of `FIXES: #`?

Reply:
Thanks for the info. My bad there, I thought `log2dch` look for `GH`. Taking into account that this PR would work as-is in that regard, I leave it open to you to support `Closes` or not.

Reply:
I went ahead and added it!